src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -700,7 +700,7 @@ export default {
           filingType: FilingTypes.ALTERATION,
           title,
           filingId: header.filingId,
-          filingAuthor: header.certifiedBy || 'Registry Staff',
```

Comment:
Maybe I should add 'Registry Staff' in the Staff Notation. I'll  check next week.

Reply:
Andre, you don't need to do anything special in your UI change. As discussed in Standup, you can leave certifiedBy as ''.